.github/workflows/dependency-check.yml

```diff
@@ -0,0 +1,46 @@
+name: dependency check
+
+on:
+  workflow_call:
+    secrets:
+      GH_ACCESS_TOKEN:
+        required: true
+      AWS_MAVEN_ACCESS_KEY_ID:
+        required: true
+      AWS_MAVEN_ACCESS_SECRET_KEY:
+        required: true
+
+jobs:
+  dependency-check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: tastyworks/boulangerie-shared-github-workflows/.github/common-setup@main
+        with:
+          GH_ACCESS_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
+
+      - name: Run Gradle dependencyCheckAnalyze
+        working-directory: ./project
+        run: >
+          ./gradlew
+          -PAWS_MAVEN_ACCESS_KEY_ID=${{ secrets.AWS_MAVEN_ACCESS_KEY_ID }}
+          -PAWS_MAVEN_ACCESS_SECRET_KEY=${{ secrets.AWS_MAVEN_ACCESS_SECRET_KEY }}
+          dependencyCheckAnalyze --info
```

Comment:
You can drop the `--info` flag now since we figured out the issue to reduce the logging output